src/foam/nanos/cron/CronSchedule.js

```diff
@@ -52,12 +56,13 @@ foam.CLASS({
     {
       class: 'Int',
       name: 'second',
-      value: 0,
+      value: -1,
```

Comment:
does this contradict with min, max?

Reply:
please elaborate.

Reply:
managed to revert default seconds to 0.

Reply:
We still have some cron jobs with second set to -1. Did this mean that the cron should run every second? In any case, don't we need something like javaPreset to adjust the second when it was set to -1 already?

Reply:
we don't want anything running every second and I'm sure those same jobs have some other specifier set. So we are not supporting every second.

Reply:
Yes those crons have other time specifiers set. As long as you are okay with them having second set to -1 (though it doesn't mean anything and contradict with min max on second property), the pr looks good to me

Reply:
Thanks. Yes, -1 is ignored in getNextScheduledTime.